pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -208,14 +209,26 @@ func createLogicalReplicationStreamPlanHook(
 			return err
 		}
 
+		sourceTypes := make([]*descpb.TypeDescriptor, len(spec.TypeDescriptors))
+		for i, desc := range spec.TypeDescriptors {
+			sourceTypes[i] = &desc
+		}
+		// TODO(rafi): do we need a different type resolver?
```

Comment:
if the user is using the sql path, we still need to fast fail here until #132164 is resolved.

Reply:
done